src/components/organisms/MessageBox/MessageBox.tsx

```diff
@@ -36,7 +38,7 @@ const MessageBox: React.FC = () => {
       // @ts-ignore
       notification[notificationType]({
         key: alert.id,
-        message: alert.title,
+        message: enhanceErrorMessage(alert.title),
         description: <NotificationMarkdown notification={alert} type={notificationType} />,
```

Comment:
I would put here instead of `notification={alert}` -> `notification={{...alert, message: enhanceErrorMessage(alert.message)}}`, so in this way you don't have to modifiy anything inside of the `NotificationMarkdown` component